app/views/families/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<p class="title"><%= t(".title") %></p>
+<div class="container">
+  <%= simple_form_for @family do |f| %>
```

Comment:
You should pull this form into a `_form.html.erb` partial and reuse it in both the new and edit views.